user.js

```diff
@@ -0,0 +1,37 @@
+user_pref("browser.shell.checkDefaultBrowser", false);
```

Comment:
@karansapolia Looks like this slipped in on accident, can you remove it?

Reply:
sorry, I corrected that!